frontend/src/metabase/static-viz/components/XYChart/XYChart.tsx

```diff
@@ -62,7 +62,14 @@ export const XYChart = ({
     series = calculateStackedItems(series);
   }
 
-  const yDomains = calculateYDomains(series, settings.goal?.value);
+  const setMinValue = settings.visualization_settings["graph.y_axis.min"];
```

Comment:
I was thinking about this yesterday, but forgot to comment. `setMinValue` sounds more like a function than a value. I know `minValue` also doesn't convey the exact same thing. We use `minValue` in other places (viz) to say it's the min value from the series, not the settings.

So I'm not totally confident about the suggestion, but something along the line of `minValueSetting` should do. What do you think?

Reply:
Yea, that's a good call. thanks :)